core/src/main/java/cz/xtf/core/helm/HelmClients.java

```diff
@@ -0,0 +1,108 @@
+package cz.xtf.core.helm;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Paths;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.SystemUtils;
+
+import cz.xtf.core.config.OpenShiftConfig;
+import cz.xtf.core.config.XTFConfig;
+import cz.xtf.core.http.Https;
+import cz.xtf.core.openshift.CLIUtils;
+
+public class HelmClients {
```

Comment:
Could you please update this to follow the same pattern as oc binary.

public OpenShiftBinaryManagerFactory + package-private OpenShiftBinaryManager + path resolver(s)

it would be thread-safe and testable (also please add tests)